styles/variables-base.less

```diff
@@ -336,21 +336,28 @@
 // Popup
 // ---------------------------------------
 @agate-popup-min-width: 510px;
-@agate-popup-max-width: 70%;
```

Comment:
Why did you remove the @agate-popup-max-width variable? It will break our UX.

Reply:
I saw it had no effect on the Popup on all skins

Reply:
If adding `style={{width: '100%'}}` in the Popup sampler
```
storiesOf('Agate', module)
	.add(
		'Popup',
		() => (
			<div>
				<Popup
					...
					style={{width: '100%'}}
				>
```

in `develop` branch
![image](https://user-images.githubusercontent.com/4239873/86576553-d7469600-bfb3-11ea-8cb4-4f5ffdf0acde.png)

in your branch
![image](https://user-images.githubusercontent.com/4239873/86576441-b3835000-bfb3-11ea-87fe-4491ca080bd8.png)

The popup width could be resized by app developers. The GUI developers wanted not to expand the popup width to the same size with the screen width. So that's the reason why we added the max-width.

Reply:
ok, I reverted the change :)

Reply:
reverted this in variables-base but kept the removal in variables-gallium and variables-silicon as they had the same value